doc/installationguide.md

```diff
@@ -254,6 +254,9 @@ overrides.
 | IOTA_MONGO_PORT           | `mongodb.port`                  |
 | IOTA_MONGO_DB             | `mongodb.db`                    |
 | IOTA_MONGO_REPLICASET     | `mongodb.replicaSet`            |
+| IOTA_MONGO_USER 	    | `mongodb.user`		      |
```

Comment:
Myabe you could add some information about the new parameters in the `mongodb` section above (around line 136 in this same file).